### ISSUE TITLE
How about the time complexity of LFTM

### ISSUE THREAD

**Issue by NobodyWHU** [datquocnguyen/LFTM#1]
I am interested in LFTM, and I find this model cost a lot time compared with other topic model. So I want to know the time complexity of this method. 
Thanks.

**Comment by datquocnguyen** [comment]
As shown in the paper, the inference algorithms for LF-LDA and LF-DMM interleave a MAP estimation for learning topic vectors in each Gibbs sampling iteration, thus LF-LDA and LF-DMM are slower than LDA and DMM. 

For learning a topic vector (here, topic vectors can be learned in parallel), the complexity of a single gradient update is `O(2 * V * k)` where `V` is the vocabulary size and `k` is the number of vector dimensions. Let `n` be the averaged number of gradient updates, we have  `O(2 * V * k * n)`.

To obtain a faster training process, you might want to try with small `k` value, e.g., using the pre-trained 50-dimensional word vectors from [GloVe](http://nlp.stanford.edu/projects/glove/).

**Comment by NobodyWHU** [comment]
Thank you for your explanation！
